setup.py

```diff
@@ -30,12 +34,16 @@ def get_version():
     'pytest-asyncio>=0.5.0',
     'pytest-cov>=2.3.1',
     'logbook>=1.0.0',
+    'flake8>=3.0.4',
+    'isort>=4.2.5',
+    'collective.checkdocs>=0.2',
+    'Pygments>=2.1.3'  # required by checkdocs
```

Comment:
Doesn't this work?

    Pygments>=2.1.3,<2.3

Then you'll get the latest version that is guaranteed to work according to semver. If you feel adventurous, you could also use `Pygments>=2.1.3,<3`, but that is bound to break some time in the future. (Maybe not with the pygments dependency, but with others :))